src/internal/views/nunjucks/billing/macros/batch-metadata.njk

```diff
@@ -0,0 +1,18 @@
+{% macro batchMetadata(batch) %}
+    <div class="govuk-grid-column-full">
+      <dl class="meta govuk-!-margin-bottom-3">
+        <div class="meta__row">
+          <dt class="meta__key black-text">Date created</dt>
```

Comment:
Could this be replaced with a modifier on meta__key. I don't know what the black is trying to convey, but something like `meta__key--loud`.